picard/ui/mainwindow.py

```diff
@@ -189,7 +189,8 @@ def keyPressEvent(self, event):
     def show(self):
         self.restoreWindowState()
         super().show()
-        self.auto_update_check()
+        if QtWidgets.QApplication.instance().autoupdate_enabled:
```

Comment:
I believe that the value is exposed as `self.tagger.autoupdate_enabled` so the `instance()` object shouldn't be required.  Same throughout the file.

Reply:
Thanks, I didn't notice that.